python/lsst/ts/standardscripts/auxtel/daytime_checkout/latiss_checkout.py

```diff
@@ -0,0 +1,133 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["LatissCheckout"]
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.auxtel.latiss import LATISS
+from ...utils import get_topic_time_utc
+
+STD_TIMEOUT = 10  # seconds
+
+
+class LatissCheckout(salobj.BaseScript):
+    """DayTime LATISS Checkout SAL Script.
+
+    This script performs the daytime checkout of
+    LATISS to ensure it is ready to be released
+    for nighttime operations
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Bias Frame Verification": Before taking bias frame.
+    - "Engineering Frame Verification": Before taking engineering frame. Final
+    checkpoint in script.
+
+    **Details**
+
+    This script is used to perform the daytime checkout of LATISS to ensure it
+    is ready for nighttime operations. It does not include telescope or dome
+    motion. It will enable LATISS, take a bias and an engineering frame, and
+    check that both frames were successfully ingested by OODS.
+    """
+
+    def __init__(self, index=1):
+
+        super().__init__(
+            index=index,
+            descr="Execute daytime checkout of LATISS.",
+        )
+
+        # Instantiate latiss. We need to do this after the call to
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.latiss = LATISS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
```

Comment:
You can just remove this one

Reply:
schema_yaml variable removed from all daytime_checkout scripts.